apps/prairielearn/src/pages/administratorQuery/administratorQuery.html.ts

```diff
@@ -0,0 +1,328 @@
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head') %>", resLocals)}
+        <link href="${nodeModulesAssetPath('highlight.js/styles/default.css')}" rel="stylesheet" />
+        <link
+          href="${nodeModulesAssetPath('tablesorter/dist/css/theme.bootstrap.min.css')}"
+          rel="stylesheet"
+        />
+        <script src="${nodeModulesAssetPath(
+            'tablesorter/dist/js/jquery.tablesorter.min.js',
+          )}"></script>
+        <script src="${nodeModulesAssetPath(
+            'tablesorter/dist/js/jquery.tablesorter.widgets.min.js',
+          )}"></script>
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar') %>", {
+          ...resLocals,
+          navPage: 'admin',
+          navSubPage: 'queries',
+        })}
+
+        <main id="content" class="container-fluid">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex align-items-center">
+              <span class="font-weight-bold text-monospace">${sqlFilename}</span>
+              <button
+                class="btn btn-xs btn-light ml-2 my-n2"
+                type="button"
+                data-toggle="collapse"
+                data-target="#sql-query"
+                aria-expanded="false"
+                aria-controls="sql-query"
+              >
+                Show SQL<i class="fas fa-caret-down"></i>
+              </button>
+              <span class="ml-3">&mdash;</span>
+              <span class="ml-3">${info.description}</span>
+            </div>
+
+            <div id="sql-query" class="collapse">
+              <pre class="m-0 p-2 bg-light border-bottom"><code class="sql">${unsafeHtml(
+                resLocals.sqlHighlighted,
+              )}</code></pre>
+            </div>
+            <div class="card-body">
+              <form name="run-query-form" method="POST">
+                ${info.params
+                  ? info.params.map((param) => {
+                      return html`
+                        <div class="form-group">
+                          <label for="${`param-${param.name}`}">${param.name}</label>
+                          <input
+                            class="form-control"
+                            type="text"
+                            id="${`param-${param.name}`}"
+                            aria-describedby="${`param-${param.name}-help`}"
+                            name="${param.name}"
+                            autocomplete="off"
+                            ${params?.[param.name]
+                              ? html`value="${params[param.name]}"`
+                              : html`value="${param.default ?? ''}"`}
+                          />
+
+                          <small id="${`param-${param.name}-help`}" class="form-text text-muted"
+                            >${param.description}</small
+                          >
```

Comment:
Sometimes you have to nudge Prettier in the right direction:

```suggestion
                          <small id="${`param-${param.name}-help`}" class="form-text text-muted">
                            ${param.description}
                          </small>
```

Reply:
Resolved in comment 2f39462.